### PR TITLE
Fixes #85

### LINKED ISSUE #85: Display test result info in XmlView
When a test has been run, the xml displayed should be that for the result rather than simply for the test exploration. The model method GetResultForTest will return it if available.

One tricky bit: the results are isolated - that is the child results are not included but must be fetched separately. This may make it harder than I originally imagined, in which case it can be delayed.

### PR DESCRIPTION
**Pull request by iCodeIT**
Show both result nodes and test nodes in the xml view.

### CODE REVIEW

**Comment by iCodeIT** [comment]
Ping @CharliePoole ;-)

File: src/nunit-gui/Presenters/XmlPresenter.cs [hunk 1]
```diff
@@ -99,5 +99,31 @@ private void DisplayXml()
...
+            if (resultNode != null)
+            {
+                currentXml = resultNode.Xml.Clone();
+            }
+            else
+            {
+                currentXml = testNode.Xml.Clone();
+                while (currentXml.ChildNodes.Count > 0)
+                {
+                    currentXml.RemoveChild(currentXml.ChildNodes[0]);
```
Comment (CharliePoole) [inline src/nunit-gui/Presenters/XmlPresenter.cs]: Here, I believe you are removing child nodes that should not be removed: properties for example, possibly element content. I think you need to have a method that removes child tests only, that is, elements with the name "test-case" or "test-suite".

**Comment by CharliePoole** [comment]
Looks generally right, with the exception of my comment above. Once you fix that I'll pull it down and try it - nothing like actually seeing the gui. :-)

**Comment by iCodeIT** [comment]
@CharliePoole  Ok, I have made the changes to the way xml-nodes are replaced. I think this should work - at least I see both property nodes and CDATA nodes now.

**Comment by CharliePoole** [comment]
Looks good to me... I'm merging.

For next time... our naming standard for branches in our own repos is like issue-85 rather than Issue85. We don't do anything automated with the gui branches yet, but we may in future.

**Pull request merged**